pages/status.tsx

```diff
@@ -199,7 +199,12 @@ const Status: FC = () => {
               id="esrf"
               name="esrf"
               label={t('esrf.label')}
-              onChange={formik.handleChange}
+              onChange={(event) =>
```

Comment:
Create a handle callback function:

```tsx
const handleOnESRFChange: ChangeEventHandler<HTMLInputElement> = useCallback(({ target }) => {
  formik.setFieldValue(target.name, target.value.replace(/[^a-z0-9]/gi, ''))
}, [formik])
```

Reply:
Had a discussion about useCallback with Stefan since I originally didn't see the need to add a handler outside of the `onChange` that used the same function, but given some examples I've looked at and the docs I think this makes sense given we're using Formik.